lib/Echidna/ABI.hs

```diff
@@ -81,7 +81,9 @@ data GenDict = GenDict { _pSynthA    :: Float
                          -- ^ Whole calls to use, sorted by type
                        , _defSeed    :: Int
                          -- ^ Default seed to use if one is not provided in EConfig
-                       } deriving (Show)
```

Comment:
No more `Show` ?

Reply:
There's no good way to show the return typing function, and we never use that instance (I added it to debug something way back when and forgot to remove it)